packages/cozy-client/src/queries/optimize.js

```diff
@@ -9,8 +9,8 @@ const isIdQuery = query => query.id || query.ids
 /**
  * Optimize queries on a single doctype
  *
- * @param  {[QueryDefinition]} queries - Queries of a same doctype
- * @return {[QueryDefinition]} Optimized queries
+ * @param  {Array.QueryDefinition} queries - Queries of a same doctype
+ * @return {Array.QueryDefinition} Optimized queries
```

Comment:
I didn't know this syntax (Array.Type) and I cannot find it on the JSX doc. I think it should be `QueryDefinition[]`.

http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters

Reply:
I got confused by this doc : http://usejsdoc.org/tags-type.html, which is for the type definition. It seems your version is correct